#! /bin/sh /usr/share/dpatch/dpatch-run
## stable-042-3835_gnome-sharp-typo.dpatch by Tim Retout <tim@retout.co.uk>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch from upstream stable branch to fix a configure.in typo.

@DPATCH@
diff -urNad f-spot~/ChangeLog f-spot/ChangeLog
--- f-spot~/ChangeLog	2008-04-15 23:32:08.000000000 +0100
+++ f-spot/ChangeLog	2008-04-15 23:32:54.000000000 +0100
@@ -1,5 +1,9 @@
 2008-04-15  Stephane Delcroix  <sdelcroix@novell.com>
 
+	backporting r3766 to the 042_STABLE branch
+
+2008-04-15  Stephane Delcroix  <sdelcroix@novell.com>
+
 	backporting r3832 to the 042_STABLE branch
 
 2008-03-25  Stephane Delcroix  <sdelcroix@novell.com>
diff -urNad f-spot~/configure.in f-spot/configure.in
--- f-spot~/configure.in	2008-04-15 23:32:08.000000000 +0100
+++ f-spot/configure.in	2008-04-15 23:32:08.000000000 +0100
@@ -75,7 +75,7 @@
 
 CSC_DEFINES=""
 PKG_CHECK_MODULES(GNOME_SHARP, gnome-sharp-2.0 >= 2.8)
-if pkg-config --atleast-version=2.19.90 gnome-sharp; then
+if pkg-config --atleast-version=2.19.90 gnome-sharp-2.0; then
 	PKG_CHECK_MODULES(GTKHTML_SHARP, gtkhtml-sharp-3.14 >= 2.19.90)
  	GTKHTML_SHARP_PKG="-pkg:gtkhtml-sharp-3.14 -pkg:gnome-sharp-2.0"
  	CSC_DEFINES="$CSC_DEFINES -d:GTKHTML_3_14"
